lib/streamlit/elements/spinner.py

```diff
@@ -82,4 +82,9 @@ def set_message():
                 display_message = False
         with legacy_caching.suppress_cached_st_function_warning():
             with caching.suppress_cached_st_function_warning():
-                message.empty()
+                # TODO(lukasmasuch): Set the spinner element to an empty container instead
```

Comment:
Why is this a TODO comment? Seems like it's already done!

Reply:
Update the comment 👍